content/en/docs/collector/configuration.md

```diff
@@ -546,21 +546,30 @@ service:
 
 ### Telemetry
 
-Telemetry is where the telemetry for the collector itself can be configured. It
+Telemetry is where the telemetry for the Collector itself can be configured. It
 has two subsections: `logs` and `metrics`.
 
 The `logs` subsection allows configuration of the logs generated by the
-collector. By default the collector will write its logs to stderr with a log
-level of `INFO`. You can also add static key-value pairs to all logs using the
+Collector.
+
+By default, the Collector will write its logs to `stderr` with a log level of
+`INFO`. You can also add static key-value pairs to all logs using the
 `initial_fields` section.
-[View the full list of `logs` options here.](https://github.com/open-telemetry/opentelemetry-collector/blob/7666eb04c30e5cfd750db9969fe507562598f0ae/config/service.go#L41-L97)
+
+View the full list of the `logs` configuration options
+[here](https://github.com/open-telemetry/opentelemetry-collector/blob/7666eb04c30e5cfd750db9969fe507562598f0ae/config/service.go#L41-L97).
 
 The `metrics` subsection allows configuration of the metrics generated by the
-collector. By default the collector will generate basic metrics about itself and
-expose them for scraping at `localhost:8888/metrics`
-[View the full list of `metrics` options here.](https://github.com/open-telemetry/opentelemetry-collector/blob/7666eb04c30e5cfd750db9969fe507562598f0ae/config/service.go#L99-L111)
+Collector.
+
+By default, the Collector will generate basic metrics about itself and its
+components, and expose them on port `8888` and path `/metrics`, in Prometheus
+format, so that we can scrape them at `http://localhost:8888/metrics`.
 
-The following is an example telemetry configuration:
+View the full list of the `metrics` configuration options
+[here](https://github.com/open-telemetry/opentelemetry-collector/blob/7666eb04c30e5cfd750db9969fe507562598f0ae/config/service.go#L99-L111).
```

Comment:
I know that this was part of the text before, but I'd suggest dropping this link into a repo source file pinned at a specific commit. That is brittle. Besides, there are only two config options, both of which are illustrated in this page.

```suggestion
```

Reply:
Do you think it's a good idea to exact these options out for both `logs` and `metrics` sections from the code and make them part of the doc with the necessary explanation?

Reply:
@jpkrohling @brightzheng100 can this conversation be resolved?